jina/drivers/encode.py

```diff
@@ -15,14 +15,15 @@ class BaseEncodeDriver(BaseExecutableDriver):
 
     def __init__(self, executor: str = None, method: str = 'encode', *args, **kwargs):
         super().__init__(executor, method, *args, **kwargs)
+        self._use_tree_traversal = True
 
 
 class EncodeDriver(BaseEncodeDriver):
     """Extract the chunk-level content from documents and call executor and do encoding
     """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, traversal_paths: List[str] = ['r'], *args, **kwargs):
```

Comment:
Since before, the default value was implicitly the same as in the `BaseRecursiveDriver`, I would not overwrite the value here and check if all tests pass afterwards. Just remove the `traversal_path` here and in the `super().__init__(...)` call.

Reply:
Done